clouddq/runners/dbt/dbt_connection_configs.py

```diff
@@ -81,7 +81,7 @@ class GcpDbtConnectionConfig(DbtConnectionConfig):
     connection_method: DbtBigQueryConnectionMethod = None
     gcp_service_account_key_path: Optional[str] = None
     service_account_gcp_impersonation_credentials: Optional[str] = None
-    threads: int = 1
+    threads: int = 4
```

Comment:
Just as a curiosity, I found in the dbt documentation, that the recommended num of threads is 4, as this change goes. But was there a reason to have a single thread only in the previous state?